client/header/activity-panel/panels/help.js

```diff
@@ -0,0 +1,290 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { __experimentalText as Text } from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+import { Fragment, useEffect } from '@wordpress/element';
+import { Icon, chevronRight, page } from '@wordpress/icons';
+import { partial } from 'lodash';
+
+/**
+ * WooCommerce dependencies
+ */
+import { getSetting } from '@woocommerce/wc-admin-settings';
+import { List, Section } from '@woocommerce/components';
+import {
+	ONBOARDING_STORE_NAME,
+	PLUGINS_STORE_NAME,
+	SETTINGS_STORE_NAME,
+} from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import ActivityHeader from '../activity-header';
+import { getCountryCode } from 'dashboard/utils';
+import { recordEvent } from 'lib/tracks';
+import { getPaymentMethods } from 'task-list/tasks/payments/methods';
+import { compose } from 'redux';
+
+function getAppearanceItems() {
```

Comment:
Jeff what do you think about adding filters on all three of these methods? This would allow for 3PD to contextually insert help links on these setup tasks from within their extensions. It would also allow us to tweak things as needed on the ecommerce plan.

Reply:
Done in d12ea084.